src/portal/src/i18n/lang/en-us-lang.json

```diff
@@ -45,7 +45,10 @@
         "UPLOAD": "Upload",
         "NO_FILE": "No file selected",
         "ADD": "ADD",
-        "RUN": "RUN"
+        "RUN": "RUN",
```

Comment:
Please add new translations to other language files (English is enough for the content)

Reply:
fine